source/grid/grid_tools.cc

```diff
@@ -2662,7 +2662,7 @@ namespace GridTools
     static inline std::bitset<3> lookup (const MATCH_T &)
     {
       // The 1D case is trivial
-      return 4; // [true ,false,false]
+      return 1; // [true ,false,false]
```

Comment:
Indeed, this should not have been there. I guess the reason is that there are neither tests nor instatiations for 1D. Well, there are also not too many cases possible in 1D. I will address that in a separate commit so @tamiko can have a look.